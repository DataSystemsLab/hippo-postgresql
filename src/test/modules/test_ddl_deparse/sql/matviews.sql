--
-- Materialized views
--

CREATE MATERIALIZED VIEW pg_class_mv AS
  SELECT * FROM pg_class LIMIT 1 WITH NO DATA;

REFRESH MATERIALIZED VIEW pg_class_mv;
