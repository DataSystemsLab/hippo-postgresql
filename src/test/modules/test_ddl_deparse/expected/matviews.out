--
-- Materialized views
--
CREATE MATERIALIZED VIEW pg_class_mv AS
  SELECT * FROM pg_class LIMIT 1 WITH NO DATA;
NOTICE:  DDL test: type simple, tag CREATE MATERIALIZED VIEW
REFRESH MATERIALIZED VIEW pg_class_mv;
NOTICE:  DDL test: type simple, tag REFRESH MATERIALIZED VIEW
